vcd/resource_vcd_snat_test.go

```diff
@@ -167,8 +166,8 @@ func TestAccVcdSNAT_BackCompability(t *testing.T) {
 		CheckDestroy: testAccCheckVcdSNATDestroyForBackCompability,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config:      configText,
-				ExpectError: regexp.MustCompile(`After applying this step and refreshing, the plan was not empty:`),
+				Config: configText,
+				//ExpectError: regexp.MustCompile(`After applying this step and refreshing, the plan was not empty:`),
```

Comment:
Why are we commenting this out?

Reply:
Cause it started not to throw this exception, but left if it may return

Reply:
This is a known issue. It should be fixed in Terraform 0.12, but I've seen it in several places from time to time.

Reply:
Maybe add a TODO to remove with Terraform 0.12 support?

Reply:
added